indico/modules/rb_new/client/js/common/timeline/TimelineContent.module.scss

```diff
@@ -156,3 +153,19 @@ $split-row-height: $row-height/2;
     overflow-x: hidden;
     overflow-y: auto;
 }
+
+
+.daily-timeline {
+    margin-right: 2em;
```

Comment:
Why do you need it here? This margin makes the scrollbar e.g. in the `UnavailableRoomsModal` look a bit out of place.

Reply:
Good point, it's not useful anymore